vscode/quint-vscode/scripts/prepare-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+QUINT_VERSION=0.5.2
+
+function pause(){
+   read -p "$*"
+}
+
+git checkout main;
+git pull origin main;
+echo "Ensure that the VSCode version was bumped."
+echo "Extension: $(cat package.json | jq .version | xargs)."
+echo "Server: $(cat server/package.json | jq .version | xargs)."
+echo "You can run npm version x.y.z to bump it. Remember to bump the server as well."
```

Comment:
can we use `npm version <major|minor|patch>` rather than supplying the whole version? seems like it might reduce chance of errors. WDYT?

Reply:
TBH I still don't understand what that means. I've read this in your checklist and on `npm version` man page but didn't understand how to use it.

Reply:
`npm version patch` will increment just the patch version, `npm version minor` will bump the minor version.